src/Features/CSharp/Portable/QuickInfo/CSharpSyntacticQuickInfoProvider.cs

```diff
@@ -56,11 +59,12 @@ public CSharpSyntacticQuickInfoProvider()
             {
                 MarkInterestedSpanNearbyScopeBlock(parent, openBrace, ref spanStart, ref spanEnd);
             }
-            // If the parent is a child of a property/method declaration, object/array creation, or control flow node..
+            // If the parent is a child of a property/method declaration, object/array creation, or control flow node,
             // then walk up one higher so we can show more useful context
             else if (parent.GetFirstToken() == openBrace)
             {
-                spanStart = parent.Parent.SpanStart;
+                // parent.Parenet must be non-null, because for GetFirstToken() to have returned something it would have had to walk up to its parent
```

Comment:
nit: "Parenet"

Reply:
I fixed this locally and then failed to push it. :frowning: